snapshot/snapshot.go

```diff
@@ -532,6 +532,7 @@ func (o *snapshotter) workPath(id string) string {
 	return filepath.Join(o.root, "snapshots", id, "work")
 }
 
+// The first return value is a snapshots.Info containing only the labels, so info.Labels is the only meaningful field
```

Comment:
Why other fields is not available?

Reply:
what field is ever filled is decided by slice []Opt.  We should not make any assumption on what []opt contains